conf/addons/nvidia-device-plugin.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: nvidia-device-plugin
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      # Mark this pod as a critical add-on; when enabled, the critical add-on scheduler
+      # reserves resources for critical add-on pods so that they can be rescheduled after
+      # a failure.  This annotation works in tandem with the toleration below.
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+      labels:
+        name: nvidia-device-plugin
+    spec:
+      nodeSelector:
+        kops.k8s.io/instancegroup: gpu-nodes
+      tolerations:
+      # Allow this pod to be rescheduled while the node is in "critical add-ons only" mode.
+      # This, along with the annotation above marks this pod as a critical add-on.
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - key: nvidia.com/gpu
+        effect: NoSchedule
+      containers:
+      - image: nvidia/k8s-device-plugin:1.9
```

Comment:
not sure if this image will ever change, but maybe
{{ getenv "NVIDIA_DEVICE_PLUGIN_IMAGE" | default "nvidia/k8s-device-plugin:1.9" }}